docker/actinia-core-tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mundialis/actinia-core:latest as actinia_test
+FROM mundialis/actinia-core:4.14.1 as actinia_test
```

Comment:
do you have to adjust this with every release?

Reply:
Yes and no - renovate will take care of it.